sretoolbox/container/image.py

```diff
@@ -133,6 +140,11 @@ def __init__(self, url, tag_override=None, username=None, password=None,
         self._cache_manifest = None
         self._cache_content_type = None
 
+    def _can_response_be_cached(self):
+        # Determines if we have a method to handle response cache entries for
```

Comment:
Nit: method doc should use """ instead of #. Same applies for other methods below

Reply:
I didn't want to put a proper docstring in a private method but I wanted to document it. I had the same doubt and this is what I got from PEP8: https://peps.python.org/pep-0008/#documentation-strings

_Write docstrings for all public modules, functions, classes, and methods. Docstrings are not necessary for non-public methods, but you should have a comment that describes what the method does. This comment should appear after the def line._

Reply:
Ok, I wasn't aware of that. TIL